src/lib/util/RateLimit.js

```diff
@@ -6,7 +6,7 @@ class RateLimit {
 	/**
 	 * @since 0.5.0
 	 * @param {number} bucket The number of requests before this is limited
-	 * @param {number} cooldown The amount of time for each bucket
+	 * @param {number} cooldown The amount of time in milliseconds for this to reset
```

Comment:
Wouldn't it be better wording to say "the amount of milliseconds for the cooldown to expire"? (same for the other occurences)